Frontend/ui-customer/src/Components/DriverInfoCard/View.purs

```diff
@@ -1175,11 +1175,11 @@ configurations =
               , paddingOTP : Padding 11 0 11 7
               }
 
-getVehicleImage :: String -> String 
-getVehicleImage variant = do
+getVehicleImage :: String -> String -> String
+getVehicleImage variant detail = do
```

Comment:
can you define proper argument name instead of `detail`

Reply:
done